release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -195,9 +195,47 @@ The {product-title} internal registry and image streams now support Open Contain
 
 //Add link
 
-[id="ocp-4-7-operators"]
+[id="ocp-4-7-olm"]
 === Operator lifecycle
 
+[id="ocp-4-7-safe-operator-upgrades"]
+==== Safe Operator upgrades
+
+To make upgrades more robust, it is recommend that Operators actively communicate with the service that is about to be updated. If a service is processing a critical operation, such as live migrating virtual machines (VMs) in OpenShift Virtualization or restoring a database, it might be unsafe to upgrade the related Operator at that time.
+
+In {product-title} 4.7, Operators can take advantage of the new `OperatorCondition` resource to communicate a non-upgradeable state to Operator Lifecycle Manager (OLM), for example if a related service is performing a critical operation. The non-upgradeable state delays any pending Operator upgrade, whether automatically or manually approved, until the Operator finishes the operation and reports upgrade readiness.
+
+See _Operator conditions_ for more about how OLM uses this communication channel.
+
+// Will update to an xref after related PR merge
+
+See _Managing Operator conditions_ for details on overriding states in OLM as a cluster administrator.
+
+// Will update to an xref after related PR merge
+
+See xref:../operators/operator_sdk/osdk-generating-csvs.adoc#osdk-operatorconditions_osdk-generating-csvs[Enabling Operator conditions] for details on updating your project as an Operator developer to use the communication channel.
+
+[id="ocp-4-7-osdk"]
+=== Operator development
+
+[id="ocp-4-7-osdk-supported"]
+==== Operator SDK now fully supported
+
+As of {product-title} 4.7, the Operator SDK is now a fully supported Red Hat offering. With the downstream release of Operator SDK v1.3.0, officially supported and branded Operator SDK tooling is now available for download directly from Red Hat.
+
+The Operator SDK CLI assists Operator developers and independent software vendor (ISV) partners in writing Operators that provide a great user experience and are compatible with OpenShift and Operator Lifecycle Manager (OLM).
```

Comment:
s/OpenShift/{product-title}

Reply:
I'll change this to "OpenShift distributions". Or maybe "variants."